PKG_SOURCE_LOCAL override is broken

#### Description
using a local copy for a package with `PKG_SOURCE_LOCAL` is broken since #13036 was merged. Now the packages source is copied from the local path, but it is not build anymore.

This should be fixed ASAP, as this is stalling my ongoing experiments :-)

#### Steps to reproduce the issue
Take any package (e.g. `nimble`) and try to build RIOT with a local copy of that package
```
PKG_SOURCE_LOCAL_NIMBLE=/local/path/foo make clean all
```
You will see in the build output, that the package is actually copied:
```
Building application "nimble_gatt" for "nrf52dk" with MCU "nrf52".

rm -Rf /home/hauke/dev/riot/RIOT/examples/nimble_gatt/bin/pkg/nrf52dk/nimble
mkdir -p $(dirname /home/hauke/dev/riot/RIOT/examples/nimble_gatt/bin/pkg/nrf52dk/nimble)
cp -a /home/hauke/dev/ble/mynewt-nimble /home/hauke/dev/riot/RIOT/examples/nimble_gatt/bin/pkg/nrf52dk/nimble
touch /home/hauke/dev/riot/RIOT/examples/nimble_gatt/bin/pkg/nrf52dk/nimble/.prepared
"make" -C /home/hauke/dev/riot/RIOT/pkg/nimble
"make" -C /home/hauke/dev/riot/RIOT/boards/nrf52dk
"make" -C /home/hauke/dev/riot/RIOT/boards/common/nrf52xxxdk
"make" -C /home/hauke/dev/riot/RIOT/core
...
```
but it is not compiled
```
...
"make" -C /home/hauke/dev/riot/RIOT/sys/xtimer
arm-none-eabi-gcc: error: /home/hauke/dev/riot/RIOT/examples/nimble_gatt/bin/nrf52dk/nimble_controller.a: No such file or directory
arm-none-eabi-gcc: error: /home/hauke/dev/riot/RIOT/examples/nimble_gatt/bin/nrf52dk/nimble_drivers_nrf5x.a: No such file or directory
arm-none-eabi-gcc: error: /home/hauke/dev/riot/RIOT/examples/nimble_gatt/bin/nrf52dk/nimble_host.a: No such file or directory
arm-none-eabi-gcc: error: /home/hauke/dev/riot/RIOT/examples/nimble_gatt/bin/nrf52dk/nimble_host_store_ram.a: No such file or directory
arm-none-eabi-gcc: error: /home/hauke/dev/riot/RIOT/examples/nimble_gatt/bin/nrf52dk/nimble_host_util.a: No such file or directory
arm-none-eabi-gcc: error: /home/hauke/dev/riot/RIOT/examples/nimble_gatt/bin/nrf52dk/nimble_npl_riot.a: No such file or directory
arm-none-eabi-gcc: error: /home/hauke/dev/riot/RIOT/examples/nimble_gatt/bin/nrf52dk/nimble_porting_nimble.a: No such file or directory
arm-none-eabi-gcc: error: /home/hauke/dev/riot/RIOT/examples/nimble_gatt/bin/nrf52dk/nimble_riot_contrib.a: No such file or directory
arm-none-eabi-gcc: error: /home/hauke/dev/riot/RIOT/examples/nimble_gatt/bin/nrf52dk/nimble_svc_gap.a: No such file or directory
arm-none-eabi-gcc: error: /home/hauke/dev/riot/RIOT/examples/nimble_gatt/bin/nrf52dk/nimble_svc_gatt.a: No such file or directory
arm-none-eabi-gcc: error: /home/hauke/dev/riot/RIOT/examples/nimble_gatt/bin/nrf52dk/nimble_tinycrypt.a: No such file or directory
arm-none-eabi-gcc: error: /home/hauke/dev/riot/RIOT/examples/nimble_gatt/bin/nrf52dk/nimble_transport_ram.a: No such file or directory
/home/hauke/dev/riot/RIOT/examples/nimble_gatt/../../Makefile.include:537: recipe for target '/home/hauke/dev/riot/RIOT/examples/nimble_gatt/bin/nrf52dk/nimble_gatt.elf' failed
make: *** [/home/hauke/dev/riot/RIOT/examples/nimble_gatt/bin/nrf52dk/nimble_gatt.elf] Error 1
```


#### Expected results
Should be build using the local copy

#### Actual results
does not

--- a/pkg/local.mk
+++ b/pkg/local.mk
@@ -6,7 +6,9 @@
 #
 # WARNING: any local changes made to $(PKG_BUILDDIR) *will* get lost!
 
-.PHONY: prepare clean
+.PHONY: prepare clean all
+
+all: $(PKG_BUILDDIR)/.prepared
 
 prepare: $(PKG_BUILDDIR)/.prepared
 	@true

